code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -1175,20 +1175,18 @@
 	name = "Rally Hive"
 	action_icon_state = "rally_hive"
 	mechanics_text = "Rallies the hive to a congregate at a target location, along with an arrow pointer. Gives the Hive your current health status. 60 second cooldown."
-	use_state_flags = XACT_TARGET_SELF
 	ability_name = "rally hive"
 	plasma_cost = 0
 	keybind_signal = COMSIG_XENOABILITY_RALLY_HIVE
+	keybind_flags = XACT_KEYBIND_USE_ABILITY
 	cooldown_timer = 60 SECONDS
 
-/datum/action/xeno_action/activable/rally_hive/use_ability(atom/A)
+/datum/action/xeno_action/activable/rally_hive/use_ability()
 
 	var/mob/living/carbon/xenomorph/X = owner
 
-	X.face_atom(A) //Face towards the target so we don't look silly
-
-	xeno_message("<span class='xenoannounce'>Our leader [X] is rallying the hive to [AREACOORD_NO_Z(A)]!</span>", 3, X.hivenumber, FALSE, get_turf(A), 'sound/voice/alien_distantroar_3.ogg')
-	notify_ghosts("\ [X] is rallying the hive to [AREACOORD_NO_Z(A)]!", source = get_turf(A), action = NOTIFY_JUMP)
+	xeno_message("<span class='xenoannounce'>Our leader [X] is rallying the hive to [AREACOORD_NO_Z(X.loc)]!</span>", 3, X.hivenumber, FALSE, X, 'sound/voice/alien_distantroar_3.ogg',TRUE,null,arrow_hive_leader_rallying)
+	notify_ghosts("\ [X] is rallying the hive to [AREACOORD_NO_Z(X.loc)]!", source = X, action = NOTIFY_JUMP)
```

Comment:
why did you switch here for jumping to the xeno and not the spot theyre pointing to ?

Reply:
that's a bit beyond the scope of the PR, but i talked to surr and he's fine with it. And because pointing a spot is actually less usefull than following the leader, since the rally hive will last much longer